host_core/lib/host_core/actors/actor_module.ex

```diff
@@ -55,11 +55,47 @@ defmodule HostCore.Actors.ActorModule do
     GenServer.call(pid, :health_check)
   end
 
+  def live_update(pid, bytes, claims) do
+    GenServer.call(pid, {:live_update, bytes, claims}, 30_000)
+  end
+
   @impl true
   def init({claims, bytes, oci}) do
     start_actor(claims, bytes, oci)
   end
 
+  def handle_call({:live_update, bytes, claims}, _from, agent) do
+    Logger.info("Actor #{claims.public_key} performing live update")
+
+    imports = %{
+      wapc: Imports.wapc_imports(agent),
+      wasmbus: Imports.wasmbus_imports(agent)
+    }
+
+    # shut down the previous Wasmex instance to avoid orphaning it
+    old_instance = Agent.get(agent, fn content -> content.instance end)
+    GenServer.stop(old_instance, :normal)
+
+    {:ok, instance} = Wasmex.start_link(%{bytes: bytes, imports: imports})
```

Comment:
Doing some thinking here, so each ActorModule will receive the call individually, so if I have 100 instances of actor `MASDASDASD`, there will be 100 info logs saying that actor is performing a live update?

Reply:
I think this might be better as either a trace log, or placed in the ActorSupervisor `live_update` section